src/build.py

```diff
@@ -1103,6 +1103,7 @@ def TestBare():
       runner=os.path.join(INSTALL_BIN, 'wasm.opt'),
       indir=s2wasm_out,
       fails=SPEC_KNOWN_TORTURE_FAILURES,
+      warn_only=True,  # TODO all the spec tests fail with syntax error
```

Comment:
Oh wait this is actually because the spec hasn't been merged to 0xc yet. Once it does, then that bug will be the relevant one.
Actually we should update this PR to pin the spec to the 0xc branch, since I don't know if anyone will merge it until Andreas gets back (I'm not sure if there are any outstanding issues left in ml-proto).